Library/Core/Uno.Permissions/Permissions.java

```diff
@@ -76,7 +76,11 @@ private static void nextRequest()
             _currentRequest.requestID);
     }
 
-    public static void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
+    public static void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) 
+    {
+        if( _currentRequest != null) 
```

Comment:
To me, it looks like you want the early-out when `_currentRequest == null`.

As it stands now, the method will still crash when `_currentRequest` is null, and just do nothing when not null...

Reply:
You are right, thanks for your advice ;)

Reply:
It fixed now @mortend

Reply:
Nice :)

Reply:
Whoops, sloppy review on my end! Good catch!